spring-cloud-app-broker-deployer-cloudfoundry/src/test/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployerTest.java

```diff
@@ -208,6 +208,7 @@ void deployAppWithPropertiesInRequest() {
 			.property(CloudFoundryDeploymentProperties.HEALTHCHECK_PROPERTY_KEY, "http")
 			.property(CloudFoundryDeploymentProperties.HEALTHCHECK_HTTP_ENDPOINT_PROPERTY_KEY, "/healthcheck")
 			.property(CloudFoundryDeploymentProperties.BUILDPACKS_PROPERTY_KEY, "buildpack1,buildpack2")
+			.property(CloudFoundryDeploymentProperties.STACK_PROPERTY_KEY, "customstack")
```

Comment:
I think this needs to be a list, like we have for buildpacks, so allow for fallbacks. I.e. 'give me `mycustomfunkystack` if available, otherwise fallback to cflinuxfs4, then cflinuxfs3'

Reply:
Ah okay thanks... I guess there's a question of whether we support fallback at the appbroker or service broker level then?

Reply:
do we have to support fallback? Since we are not defaulting to anything in app-broker it will default already to what the platform states.
If the service broker implementer decides to overwrite it with a different value, then, that must exist.
Happy with including a separate story to talk about fallbacks, but I do not see it as a must yet.